web/app/js/components/DeploymentDetail.jsx

```diff
@@ -139,15 +139,15 @@ export default class Deployment extends React.Component {
         downstreamMetrics={this.state.downstreamMetrics}
         deploymentAdded={this.state.added} />,
       _.isEmpty(this.state.deployTs) ? null :
-        <StatPane
+        <ResourceMetricsOverview
           key="stat-pane"
           lastUpdated={this.state.lastUpdated}
           timeseries={this.state.deployTs} />,
       this.renderMidsection(),
       <UpstreamDownstream
         key="deploy-upstream-downstream"
         resource="deployment"
-        entity={this.state.deploy}
+        Resource={this.state.deploy}
```

Comment:
hmm, I feel like `resource` and `Resource` are easily confused, especially since they are come right after one another... what do you think? Maybe `resourceMetrics`?

Reply:
whoops this change is unintentional. bad find/replace. but yeah, I would like to change these to "resourceName" and "resourceType" to me the most descriptive